scipy/stats/_distr_params.py

```diff
@@ -1,6 +1,7 @@
 """
 Sane parameters for stats.distributions.
 """
+import numpy as np
```

Comment:
Is this needed?

Reply:
Yes because `invdistcont` needs `np.inf` and `np.nan`.

Reply:
Of course.